### DISCUSSION TITLE
如何去除印刷文本中的手写字

### DISCUSSION THREAD

**Discussion by PeiyueYu** [PaddlePaddle/PaddleOCR#13310] [Q&A] [answered]
当前是在发票场景下，会有连着印刷文本的若干手写字出现，如何可以有效得去除呢

**Comment by UserWangZz** [comment]
出现区域固定的话，mask掉即可

**Reply by PeiyueYu** [reply]
> 出现区域固定的话，mask掉即可

一般都不是固定区域，有业务人员在有些地方做个笔记啥的

**Reply by BugManL** [reply]
检测之后分类器 分类一下？

**Comment by TOMORINAONAO** [comment, marked as answer]
paddleclas中有一个PULC语种分类模型，我用它的预训练模型自己构造数据集训练了一个印刷体-手写体的分类模型，我是在检测模型返回裁剪后的文本行后调用分类模型判断是印刷体还是手写体，之后根据分类结果调用对应的印刷体识别模型或手写体识别模型。像你这种分类成手写体的直接不识别就行。
